inputs/paritytech/zombienet/flake-module.nix

```diff
@@ -0,0 +1,83 @@
+{ self, ... }: {
+  perSystem = { config, self', inputs', pkgs, system, devnetTools, ... }:
+    let
+      paritytech-zombienet-src = pkgs.fetchFromGitHub {
+        owner = "dzmitry-lahoda-forks";
+        repo = "zombienet";
+        rev = "b9f089eb55a7cb1d4c12575b8323cb9b9fab4a60";
+        hash = "sha256-EC6XKbcI+Is0RGlmC8WGjPqiFh9Ulf3bXDoVihtYqsU=";
+      };
+      paritytech-zombienet = pkgs.stdenv.mkDerivation {
+        name = "zombienet";
+        src = "${paritytech-zombienet-src}/javascript";
+        buildInputs = with pkgs; [ nodejs ];
+        nativeBuildInputs = with pkgs;
+          [
+            yarn
```

Comment:
Is it exactly needed? You use `npm` to build `zombienet`.

Reply:
they have adhoc custom build with `so` deps (rust stuff), that just does not builds/runs in nix that simple. so need to try again later

Reply:
so answers to be clear-  i tried to run build, it failed with some nasty error. so need to try again as some point.

Reply:
https://app.clickup.com/t/20465559/NX-67